src/popup/fields/account_field.js

```diff
@@ -0,0 +1,42 @@
+const View = require('ampersand-view');
+const FilteredCollection = require('ampersand-filtered-subcollection');
+const Handlebars = require('hbsfy/runtime');
+
+Handlebars.registerHelper('is_selected', function(a, b) {
```

Comment:
are these helper methods available in all templates? if yes then let's move them to a seperate file and require it before all other views.

Reply:
i'll move all helpers to a separate file